server/src/dbschema.js

```diff
@@ -94,6 +105,17 @@ exports.createTables = function () {
         if (count != 1) {
           throw new Error("Should have deleted one row");
         }
+        return db.getConnection().then(([conn, done]) => {
```

Comment:
The problem where we aren't sure if we are inserting into a table that has a head and body